themes/atomic/atomic.theme.bash

```diff
@@ -164,8 +166,8 @@ ___atomic_prompt_clock() {
 }
 
 ___atomic_prompt_battery() {
-  [ ! -e "$BASH_IT"/plugins/enabled/battery.plugin.bash ] ||
-  [ "${THEME_SHOW_BATTERY}" != "true" ] && return
+  chk=$(isFunction battery_percentage && echo yes || echo no)
```

Comment:
Please use the `command_exists` function here - you're already using it in your other changes to check for the existence of the `WMIC` command, for example here: https://github.com/Bash-it/bash-it/pull/968/files#diff-37c868aa65fc7bb8bb78c07ff270ddd4R13 in line 13.